v2/cmd/wails/internal/commands/build/build.go

```diff
@@ -111,9 +111,18 @@ func AddBuildSubcommand(app *clir.Cli, w io.Writer) {
 	windowsConsole := false
 	command.BoolFlag("windowsconsole", "Keep the console when building for Windows", &windowsConsole)
 
+	obfuscate := false
+	command.BoolFlag("obfuscate", "Obfuscate bound methods", &obfuscate)
```

Comment:
Hiya, maybe this is easier to read for non-native English speakers?

"Code obfuscation of bound Wails methods"